### PR TITLE
Transaction breadcrumb fix

### PR DESCRIPTION
**Pull request by balthazzar**
add domain filter and remove author checking for transactions button.

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@michbil please review

File: WRIO-InternetOS/js/store/center.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ module.exports = Reflux.createStore({
                     var e = document.createElement('div');
                     e.innerHTML = result.text;
                     result = scripts(e.getElementsByTagName('script'));
+                    console.log(result);
+                } else {
+                    console.log('center.js:22:err: ', err, typeof result)
```
Comment (michbil) [inline WRIO-InternetOS/js/store/center.js]: @balthazzar why to set line number explicitly, maybe better to throw exception, or something like that?

**Comment by AlexeyAnshakov** [comment]
@balthazzar please update

**Comment by balthazzar** [comment]
@michbil please review

**Comment by michbil** [comment]
@AlexeyAnshakov PR ok

**Pull request merged**